tasks.py

```diff
@@ -592,6 +592,15 @@ def pylint(context, target=None, recursive=False):
         run_command(context, command)
 
 
+@task
+def serve_docs(context):
```

Comment:
Sorry to drop in at the 11th hour (my speciality) with this. Is there any reason we can't just call this `docs` or `docs_serve` even so that it's alpha sorted preceding with "docs"?

We also have `build-and-check-docs`, which is another action for the docs, and I'm also wondering if we can't just expose `mkdocs build|serve` using arguments?

e.g. `invoke docs build` vs. `invoke docs serve`?

Reply:
I'd requested this name because most of our other invoke tasks are verb form (build, start, stop, destroy) rather than noun form.

Reply:
@jathanism https://github.com/nautobot/nautobot/pull/2820#discussion_r1024352524